components/02-elements/tabs/Tabs.vue

```diff
@@ -0,0 +1,233 @@
+<template>
+  <div
+    class="tab"
+  >
+    <template
+      v-for="{ id, tab } in getTabsWithId"
+    >
+      <button
+        :id="id"
+        :key="'tab' + tab.tabId"
+        :class="[
+          'tab__title',
+          { 'tab__title--active': tab.tabId === activeTab }
+        ]"
+        :data-tab="tab.tabId"
+        :aria-controls="tab.tabId"
+        :aria-selected="tab.tabId === activeTab"
+        :aria-expanded="tab.tabId === activeTab"
+        @click="setActiveTab(tab.tabId)"
+      >
+        {{ tab.title }}
+        <alpaca-icon
+          v-if="icon"
+          :icon="icon"
+          class="tab__icon"
+        />
+      </button>
+      <div
+        v-if="tab.tabId === activeTab"
+        :id="tab.tabId"
+        :key="'tab__content' + tab.tabId"
+        :data-content="tab.tabId"
+        :aria-labelledby="tab.titleId"
+        :aria-hidden="tab.tabId === activeTab"
+        :class="[
+          'tab__content',
+          { 'tab__content--active': tab.tabId === activeTab }
+        ]"
+      >
+        {{ tab.content }}
```

Comment:
What do you think to use the slot for tab content? Sometimes there are some reviews, ratings, tables and so on, not just plain text.

Reply:
Now we iterate inside the component, so slot is not possible to use. First, we need rewrite this component in a different way. Should I do it?